### ISSUE TITLE
some type not work with alpha.10 version

### ISSUE THREAD

**Issue by zhaochy1990** [qontu/ngx-inline-editor#90]
hi, guys
some types such as `text`, `textarea` not work under the newest version.
the `0.2.0-alpha.9` is ok, but `0.2.0-alpha.10` will not work as expected.
try and change the version in config.js
https://embed.plnkr.co/GfGHaUdbPWyg7xBmN4vg/

P.S. something might help
I reviewed your code, it seems the problem is the newest version refactored file `input-base.ts`, the constructor get state through InlineEditorService's getState() method, getState() returns cloned state which have property `value` of null, so the `showText()` method of `input-base.ts` would always render `this.state.getState().value` of value null

```typescript
// InputBase constructor
this.state = this.service.getState().clone();
this.value = this.state.getState().value;

// InlineEditorService --> get state async
constructor(
    public events: Events,
    public config?: InlineConfig,
) {
        this.subscriptions.onUpdateStateSubscription = this.onUpdateStateOfService.subscribe(
            (state: InlineEditorState) => this.state = state,
        );
}
// the getState() method of InlineEditorService
public getState(): InlineEditorState {
    return this.state.clone();
}

// InputBase method
public showText(): string {
    return this.state.isEmpty() ? this.config.empty : this.state.getState().value;
}
```

**Comment by xxxtonixxx** [comment]
Hi @zhaochy1990, thanks for your interest and help 😃 

This issue is related to https://github.com/qontu/ngx-inline-editor/pull/79#issuecomment-325008603

It's produced because I changed the `changeDetectionStrategy` of the component to `OnPush` and something isn't working properly 😢 

I hope to have time next weekend. If you want to try to resolve it, I would be disposed to help you (it would be awesome 😄)

**Comment by zhaochy1990** [comment]
yeah, I'd love to contribute. I will try this weekend

**Comment by xxxtonixxx** [comment]
I have fixed it in latest version `0.2.0-alpha.11`. 

@zhaochy1990 thanks for your hints, they were so useful! 😄 and if you could test it 👍
